index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<script src="https://kit.fontawesome.com/f7e5caa7f7.js" crossorigin="anonymous"></script>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap" rel="stylesheet">   
+<link rel="stylesheet" href="styles.css">
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="">My Logo</a>
+            <img src="images/Enabled.png" alt="">
+    </nav>
+    </header>
+    
+    <section>
+        <div>
+            <h1>I’m Mariane
+                Glad to see you!</h1>
+            <p>I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+              <h4>LET’S CONNECT</h4> 
+                <ul  class="vector">
+                    <li><a href=""><img src="images/Vector1.svg" alt=""></a></li>
+                    <li><a href=""><img src="images/Linkedin-icon.svg" alt=""></a></li>
+                    <li><a href=""><img class="vector5" src="images/vector5.svg" alt=""></a></li>
+                    <li><a href=""><img class="vector6" src="images/vector6.svg" alt=""></a></li>
+                    <li><a href=""><img class="vector7" src="images/vector7.svg" alt=""></a></li>
+                 </ul>
+            </div>
+    </section>
+       
+    <div>
+        <img class="ellipse3" src="images/Ellipse3.svg" alt="">
+        <img class="rectangle4" src="images/Rectangle4.svg" alt="">
+        <img class="ellipse8" src="images/Ellipse8.svg" alt="">
+        <img class="ellipse5" src="images/Ellipse5.svg" alt="">
+        <img class="rectangle7" src="images/Rectangle7.svg" alt="">
+        <img class="ellipse12" src="images/Ellipse12.svg" alt="">
+        <img class="ellipse13" src="images/Ellipse13.jpg" alt="">
+        <img class="ellipse15" src="images/Ellipse15.svg" alt="">
+        <img class="ellipse4" src="images/Ellipse4.svg" alt="">
+        <img class="rectangle5" src="images/Rectangle5.svg" alt="">
+        <img class="ellipse11" src="images/Ellipse11.png" alt="">
+        <img class="rectangle6" src="images/Rectangle6.png" alt="">
+    </div> 
```

Comment:
- [ ] Kindly as asked by the previous reviewer, Please ensure to include background images by using the `background-img` CSS property. You can download all these images in one image from the Figma design and set it using `background-img` CSS property.